cmd/start.go

```diff
@@ -40,6 +43,8 @@ func init() {
 	flags.StringP(evaluatorFlagName, "e", "json", "Set an evaluator e.g. json")
 	flags.StringP(serverCertPathFlagName, "c", "", "Server side tls certificate path")
 	flags.StringP(serverKeyPathFlagName, "k", "", "Server side tls key path")
+	flags.StringToStringVarP(&ProviderArgs, ProviderArgsFlagName,
```

Comment:
This should be done in the same fashion as the others if possible, otherwise this flag won't be able to fallback to environment variable if not given

Reply:
I have found no other way to assign to maps through cobra, let me know though and ill change it

Reply:
`flags.StringToStringP(`
and
`_ = viper.BindPFlag(`
and
`viper.GetStringMapString(`
ought to do it unless I've missed something